dev-lang/hy/hy-0.19.0.ebuild

```diff
@@ -17,7 +17,7 @@ KEYWORDS="~amd64 ~x86"
 IUSE="test doc"
```

Comment:
New line after SRC_URI above, so it's LICENSE/SLOT/KEYWORDS/IUSE all together

Reply:
Done